### PR TITLE
add apollo-client configuration

### PR DESCRIPTION
**Pull request by Llavrov**

### CODE REVIEW

File: apps/web-app-e2e/project.json [hunk 1]
```diff
@@ -23,6 +23,33 @@
...
+            "options": {
+                "outputDir": "dist/storybook/web-app-e2e",
+                "configDir": "apps/web-app-e2e/.storybook",
+                "uiFramework": "@storybook/react"
+            },
+            "configurations": {
+                "ci": {
+                    "quiet": true
+                }
+            }
```
Comment (CemYil03) [inline apps/web-app-e2e/project.json:52]: I don't think this belongs to this PR

File: apps/web-app-e2e/tsconfig.json [hunk 2]
```diff
@@ -6,5 +6,11 @@
...
         "types": ["cypress", "node"]
     },
-    "include": ["src/**/*.ts", "src/**/*.js", "cypress.config.ts"]
+    "include": ["src/**/*.ts", "src/**/*.js", "cypress.config.ts"],
+    "exclude": ["**/*.stories.ts", "**/*.stories.js", "**/*.stories.jsx", "**/*.stories.tsx"],
+    "references": [
+        {
+            "path": "./.storybook/tsconfig.json"
+        }
+    ]
```
Comment (CemYil03) [inline apps/web-app-e2e/tsconfig.json:15]: This doesn't belong here either

File: apps/web-app/apollo-client.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { ApolloClient, InMemoryCache, NormalizedCacheObject } from '@apollo/client';
+
+const client: ApolloClient<NormalizedCacheObject> = new ApolloClient({
+    uri: 'https://countries.trevorblades.com',
+    cache: new InMemoryCache(),
+});
+
+export default client;
```
Comment (CemYil03) [inline apps/web-app/apollo-client.ts:8]: Can you make this in one line please? Like:

```typescript
export default const client ...
```

File: apps/web-app/components/Button.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+// Button.stories.ts|tsx
```
Comment (CemYil03) [inline apps/web-app/components/Button.stories.tsx:1]: Delete this file

File: apps/web-app/components/Button.tsx [hunk 5]
```diff
@@ -0,0 +1,5 @@
+import { Button } from '@mui/material';
```
Comment (CemYil03) [inline apps/web-app/components/Button.tsx:1]: Delete this file

File: package-lock.json
Comment (CemYil03) [inline package-lock.json:1]: How is this even possible? We forbid to use npm install...
Definitely delete

File: apps/web-app/apollo-client.ts (hunk 3 above)
Comment (Llavrov) [reply apps/web-app/apollo-client.ts:8]: ts tells me, that he don't like this way(
swears that he expects - TS1109: Expression expected.

Comment (CemYil03) [reply apps/web-app/apollo-client.ts:8]: Okay, no problem then 👍

**Review by CemYil03 — CHANGES_REQUESTED** [review]
Some more comments 💬 And can you please squash your commit into a single one (git fixup)?

**Review by CemYil03 — APPROVED** [review]

**Pull request merged**